bounceview/src/main/java/com/github/hariprasanths/bounceview/CustomDialog.java

```diff
@@ -0,0 +1,243 @@
+package com.github.hariprasanths.bounceview;
+
+import ohos.agp.components.Button;
+import ohos.agp.components.Component;
+import ohos.agp.components.ComponentContainer;
+import ohos.agp.components.DirectionalLayout;
+import ohos.agp.components.Image;
+import ohos.agp.components.LayoutScatter;
+import ohos.agp.components.Text;
+import ohos.agp.window.dialog.CommonDialog;
+import ohos.app.Context;
+
+/**
+ * Custom Common Dialog.
+ */
+public class CustomDialog extends CommonDialog {
+    Component baseCustomLayout;
+    Component baseTitleLayout;
+    Component baseContentLayout;
+    Component baseButtonLayout;
+    Component baseImageButtonLayout;
+    Component mTitleComponent = null;
+    Component mContentComponent = null;
+    Component mButtonComponent = null;
+    Component mImageButtonComponent = null;
+    Context context;
+
+    /**
+     * Constructor to create Custom Common Dialog.
+     *
+     * @param context context
+     */
+    public CustomDialog(Context context) {
+        super(context);
+        this.context = context;
+        baseCustomLayout = LayoutScatter.getInstance(context)
+                .parse(ResourceTable.Layout_custom_common_dialog, null, false);
```

Comment:
please change the layout name to base_common_dialog

Reply:
done